cmake/platform/linux/mir.cmake

```diff
@@ -1,2 +1,2 @@
-set(PLATFORM_REQUIRED_DEPS OpenGl EGL Mir LibDRM)
+set(PLATFORM_REQUIRED_DEPS OpenGl EGL Mir LibDRM Xkbcommon)
```

Comment:
Changed the xkbcommon lookup so I don't forget about it - mir and wayland both have to convert xkb keysyms to kodi XBMCKey and I copied the map from Mir. We can save it for later though since it will continue to work as-is.